src/riak_exporter/exporter.py

```diff
@@ -89,9 +89,25 @@ def parse_riak_repl_stats_data(self, riak_repl_stats_data):
                     for ckey, cvalue in value[cluster].items():
                         if isinstance(cvalue, (int, float, bool)):
                             prom_value = float(cvalue)
-                            prom_str = "riak_repl_{}{{cluster=\"{}\"}} {}".format(ckey, cluster, prom_value)
+                            prom_str = "riak_repl_fullsync_coordinator_{}{{cluster=\"{}\"}} {}".format(ckey, cluster, prom_value)
                             yield prom_str
 
+            if key == "realtime_queue_stats":
+                for rkey, rvalue in value.items():
```

Comment:
Just curious, what does the r in `rkey` and `rvalue` mean? The c in `cvalue` and `ckey` I assume is for `cluster`. But then the r gets a bit confusing if it's `right-hand-value` or `realtime_queue_stats`-`value`

Reply:
Comes from `real_time` and to differentiate from the upper level for loop